tableau_blueprints/job_status.py

```diff
@@ -0,0 +1,132 @@
+from httprequest_blueprints import execute_request
+
+import tableauserverclient as TSC
+import argparse
+import sys
+import os
+import pickle
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_UNKNOWN_ERROR = 3
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_RESOURCE = 201
+EXIT_CODE_FINAL_STATUS_ERRORED = 211
+EXIT_CODE_FINAL_STATUS_CANCELLED = 212
+EXIT_CODE_STATUS_INCOMPLETE = 210
+EXIT_CODE_JOB_NOT_FOUND = 404
+EXIT_CODE_JOB_CACNELLED = 403
+EXIT_CODE_GENERIC_QUERY_JOB_ERROR = 400
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--username', dest='username', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--site-id', dest='site_id', required=True)
+    parser.add_argument('--server-url', dest='server_url', required=True)
+    parser.add_argument('--job-id', dest='job_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def authenticate_tableau(username, password, site_id, server_url):
+    """TSC library to sign in and sign out of Tableau Server and Tableau Online.
+
+    :param username:The name of the user.
+    :param password:The password of the user.
+    :param site_id: The site_id for required datasources. ex: ffc7f88a-85a7-48d5-ac03-09ef0a677280
+    :param server_url: This corresponds to the contentUrl attribute in the Tableau REST API.
+    :return: connection object
+    """
+    try:
+        tableau_auth = TSC.TableauAuth(username, password, site_id=site_id)
+        server = TSC.Server(server_url, use_server_version=True)
```

Comment:
This is different from the `download_view.py` script. Do we or don't we need the server version explicitly set?